app/uk/gov/hmrc/gform/eval/InitFormEvaluator.scala

```diff
@@ -45,6 +46,8 @@ case class InitFormEvaluator(
   ): Boolean =
     evalBooleanExpr(includeIf.booleanExpr)
 
+  lazy val now: LocalDate = LocalDate.now()
```

Comment:
Why is this `lazy`? 

I wouldn't be surprise to learn that `lazy` overhead is more heavyweight than simple reading of the system clock.